src/ResourceManager/Network/Commands.Network/Microsoft.Azure.Commands.Network.dll-Help.xml

```diff
@@ -7178,6 +7177,68 @@ Subnets           : []</dev:code>
 		</maml:alert>
 	</maml:alertSet>
 	<command:examples>
+		<command:example>
+			<maml:title>--------------------------  Example 1 Create a peering between 2 virtual networks  --------------------------</maml:title>
+			<maml:introduction>
+				<maml:paragraph>PS C:\&gt;</maml:paragraph>
+			</maml:introduction>
+			<dev:code>PS C:\&gt; $vnet1 = Get-AzureRmVirtualNetwork –ResourceGroupName “MyResourceGroup” -Name vnet1
+PS C:\&gt; $vnet2 = Get-AzureRmVirtualNetwork -ResourceGroupName “MyResourceGroup” -Name vnet2
+PS C:\&gt; Add-AzureRmVirtualNetworkPeering -Name LinkToVNet2 -VirtualNetwork “MyVirtualNetwork” -RemoteVirtualNetworkId $vnet2.id
+PS C:\&gt; Add-AzureRmVirtualNetworkPeering -Name LinkToVNet1 -VirtualNetwork “MyVirtualNetwork” -RemoteVirtualNetworkId $vnet1.id
+</dev:code>
+			<dev:remarks>
+				<maml:para>The first command gets a virtual network object called vnet1 associated with resource group “MyResourceGroup”. Similarly, the second command gets a virtual network object called vnet2 associated with the same resource group. The third command then adds a Vnet peering link from vnet1 to vnet2. This link is named LinkToVnet2. Finally, the forth command adds a link from vnet2 to vnet1 called LinkToVnet1. Note that vnet1 and vnet2 are assumed to be previously created in this example. Note that a link must be created from vnet1 to vnet2 and vice versa in order for peering to work.</maml:para>
+				<maml:para />
+				<maml:para />
+				<maml:para>Below is example output for this example:
+
+Name                             : LinkToVNet2
+Id                               : /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/provi
+                                   ders/Microsoft.Network/virtualNetworks/MyVirtualNetwork/virtualNetworkPeerings/LinkToVNet2                     
+Etag                             : W/&quot;xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx&quot;
+ResourceGroupName                : MyResourceGroup
+VirtualNetworkName               : MyVirtualNetwork
+ProvisioningState                : Succeeded
+RemoteVirtualNetwork             : {
+                                     &quot;Id&quot;: &quot;/subscriptions/XXX/resourceGroups/MyResourceGroup
+                                   /providers/Microsoft.Network/virtualNetworks/MyVirtualNetwork&quot;
+                                   }
```

Comment:
I made a fix to output the peering state, could that property also be added in the example?